ev3dev2/motor.py

```diff
@@ -1854,8 +1863,23 @@ def __init__(self, left_motor_port, right_motor_port, desc=None, motor_class=Lar
         self.right_motor = self.motors[right_motor_port]
         self.max_speed = self.left_motor.max_speed
 
-        # color sensor used by follow_line()
-        self.cs = None
+    # color sensor used by follow_line()
+    @property
+    def cs(self):
+        return self.__cs
```

Comment:
For cs and for gyro is the double leading underscore intentional or should this be the more standard single leading underscore?  I didn't realize double leading underscore had a special meaning but then read this
https://dbader.org/blog/meaning-of-underscores-in-python

I'm not sure we need the name mangling for these...I don't think we use this for any other private variables.

Reply:
I'd argue that the double underscore is the more pythonic way of keeping the private variable truly private and having accessor methods. But I agree that if this doesn't fit the conventions of the rest of the project, then we should make it consistent with a single underscore. Let me know what you prefer.

Reply:
Personally I'd say use a single underscore to be consistent within this codebase. I didn't know about the double-underscore's behavior -- Python always has some new implicit behavior to deal with! _shakes fist_

Reply:
I like sticking with single underscore for consistency